app/models/user.rb

```diff
@@ -21,6 +21,7 @@ class User < ActiveRecord::Base
 
   before_save :check_email_changed
   after_create :download_pull_requests, :estimate_skills, :download_user_organisations
+  after_update :check_24_pull_requests
```

Comment:
I think we can hook ourselves to a better place, rather than in the `after_update` callback. This would cause every `save` call to trigger email delivery actions. I think it's better to only call `check_24_pull_requests` when an actual pull request has been made - `app/controllers/pull_request_downloads_controller.rb`, in the `create` action. This might not be the best place, but I will have to check the codebase again to make sure. What do you think?